### DISCUSSION TITLE
Icy Term

### DISCUSSION THREAD

**Discussion by mkrueger** [hecrj/iced#1549] [Show and tell]
My first medium sized iced rust project is a terminal program for old school bbs systems simulating old hardware: 

https://github.com/mkrueger/icy_term

It's basically a telnet client with old school fonts. I needed that as test project if I can realise my ansi drawing tool with iced. 
This project has become a bit larger than I thought :). And iced works well overall on all platforms.

**Comment by 13r0ck** [comment]
Add this to the project showcase 😃
https://github.com/iced-rs/iced/issues/355
